src/scss/_back-to-top.scss

```diff
@@ -14,21 +17,24 @@
   -ms-transition: all 0.3s ease;
   -o-transition: all 0.3s ease;
   transition: all 0.3s ease;
-  
+
   &:hover {
-    background: rgba(0, 0, 0, 0.9);
-    color: $color-white;
+    background: $color-white;
+    color: $color-brown;
   }
 }
 
-#return-to-top i {
-  color: $color-white;
-  margin: 0;
-  position: relative;
-  left: 1em; //16px
-  top: 0.875em; //14px
-  font-size: 1.125em; //18px
-  -webkit-transition: all 0.3s ease;
+#owl-btn {
+  z-index: 3;
+  position: fixed;
+  bottom: 4em; //24px
+  right: 1.5em; //24px
+  display: block;
+  height: 8em;
+  width: auto;
+  text-decoration: none;
+  display: none;
+  -webkit-transition: all 0.3s linear;
```

Comment:
It looks like this project is including a version of autoprefixer. Can you leave out these prefixes?